### PR TITLE
Announcement: contrib-repo

### PR DESCRIPTION
**Pull request by marmarek**

### CODE REVIEW

File: 2020-10-05-contrib-repo.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: Qubes contrib packages repository available
+categories: announcements
+author: Marek Marczykowski-Górecki
+---
+
+We are happy to announce the availability of contributed packages repository. This is the place where our community can publish Qubes OS related packages, additions and various customizations. On the other hand we provide the infrastructure and review process to make them available easily and safely in standard Qubes installation.
+
+The infrastructure was build by Frédéric Pierret, based on similar setup for building official packages. This means the same build security measurements are taken, including keeping the signing keys separate in a dedicated VM, downloading packages over Tor, publishing build logs in a non-spoofable way and few more. Frédéric is also maintainer of QubesOS-contrib repositories.
```
Comment (fepitre) [inline 2020-10-05-contrib-repo.md:10]: A typo: "was built".

**Comment by fepitre** [comment]
While you are here, maybe update travis to bionic?

**Comment by marmarek** [comment]
> While you are here, maybe update travis to bionic?

In a separate PR, sure.

**Comment by andrewdavidwong** [comment]
Revisioin pushed. Let me know what you think of the changes, @marmarek.

Good to go on my end.

**Review by andrewdavidwong — APPROVED** [review]

**Comment by marmarek** [comment]
Thanks, looks good!

**Pull request merged**